sass/settings/_sizes.sass

```diff
@@ -4,40 +4,40 @@ $ratio: 1.5
 // Variants ------------------------------------------------------------------
 
 $norm         : 1
-$xxxsmall     : ms(-4,$norm)
-$xxsmall      : ms(-3,$norm)
-$xsmall       : ms(-2,$norm)
-$small        : ms(-1,$norm)
-$small-norm   : (ms(-1,$norm) + $norm)/2
-$large-norm   : (ms(1,$norm) + $norm)/2
-$med          : ms(1,$norm)
-$large        : ms(2,$norm)
-$xlarge       : ms(3,$norm)
-$xxlarge      : ms(5,$norm)
-$xxxlarge     : ms(8,$norm)
+$xxxsmall     : ms(-4,$norm,$ratio)
```

Comment:
I had to make these changes because compass was unable to compile the file otherwise. I was getting the error:

`error sass/screen.sass (Line 7 of sass/settings/_sizes.sass: Function ms is missing argument $ratio.)`

I am not sure whether the value I supplied is correct or not though. Using `$ratio` as set above was just a guess, but the site seems to look the same.